### PR TITLE
ROC 1318 | Get credentials

### PR DESCRIPTION
**Pull request by nirajay5**
* Get access token and site ids from database
* Create a task in the queue for each site id

### CODE REVIEW

File: app/services/sentinelOne.service.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                statusCode: responseCodes.PAGE_NOT_FOUND
+            }
+            logger.error(
+                'No site ids found for this account id.',
+                { ...loggerMeta, function: 'getSentinelOneCredentials', error, data: { accountId: credential.account_id } }
+            )
+            await observationService.observationErrorEvent(credential, rabbitChannel, error, logger)
+            continue
+        }
+        let accessToken = null
```
Comment (nirajay5) [inline app/services/sentinelOne.service.js:36]: Rename to api token.

Comment (nirajay5) [reply app/services/sentinelOne.service.js:36]: Make sure this isn't being printed in the logs.

File: app/controllers/sentinelOne.controller.js [hunk 2]
```diff
@@ -1,12 +1,47 @@
...
+    await observationService.observationErrorEvent(data, rabbitChannel, ret.error, logger)
+    return false
+  }
+  const allPromises = []
+  ret.data.forEach(credential => {
+    const promises = credential.sites.map(site => {
+      const message = {
+        type: enums.queueTypes.SITE,
+        data: {
+          accessToken: credential.accessToken,
```
Comment (nirajay5) [inline app/controllers/sentinelOne.controller.js:30]: api token

File: app/utils/postgresFormatter.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+const loggerMeta = {
+    file: 'postgresFormatter.js',
+    function: null
+}
+
+const createPostgresValuesList = (total, startIdx = 1, logger) => {
```
Comment (nirajay5) [inline app/utils/postgresFormatter.js:6]: move to databaseUtils.js file

**Review by tony-davila-rocketcyber — APPROVED** [review]

**Pull request merged**